canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -265,6 +280,13 @@ class DropDown extends React.Component {
 					disabled={this.props.state === STATES.DISABLED || this.disableEmptyListDropdown}
 					type="default"
 					items={dropDown.options}
+					itemToString={(item) => (item ? (
+						<div className="properties-dropdown-label">
+							<div>{ item.label }</div>
+							<div className="custom-icon">{this.getItemIcon()}</div>
+						</div>
+					) : ""
```

Comment:
Does this fallback to the label in this case?  It shouldn't show the raw value.

Reply:
I ended up using item.label, since item.value failed multiple tests